app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -162,7 +122,15 @@ object TelemetryWrapper {
 
     @UiThread // via TelemetryHomeTileUniqueClickPerSessionCounter
     fun stopSession(context: Context) {
-        TelemetryHolder.get().recordSessionEnd()
+        fun sendEndSessionFailedEvent() {
+            // In rare instances, we have seen stopSession called before startSession.
+            // It is unknown at this time how this happens, but it is very infrequent
+            // so we are logging the event and otherwise dropping the error
+            // See https://bugzilla.mozilla.org/show_bug.cgi?id=1405192
+            TelemetryEvent.create(Category.ERROR, Method.ONPAUSE_CALLED_BEFORE_ONRESUME, Object.APP).queue()
```

Comment:
Alternatively, I think we can record events in Sentry that include a stack trace.

Reply:
@liuche do you know how we can do that?

Reply:
You'll probably have to look in the Sentry docs and potentially try out some methods to see which just send String text and which might send stacktraces.